t/08-autotest.t

```diff
@@ -338,6 +338,7 @@ subtest 'load test successfully when CASEDIR is a relative path' => sub {
     loadtest 'start';
 };
 
+pass autotest::loadtest('tests/test.py'), 'can load python test module at all';
```

Comment:
https://progress.opensuse.org/issues/99675
```
Useless use of a constant ("can load python test module at a"...) in void context at ./08-autotest.t line 342.
```

Reply:
Looks like it should have been `ok`.

Reply:
Yeah, I tried that, but that fails for me locally and in CI

Reply:
Then it should possibly use `Test::Exception::lives_ok`.

Reply:
`ok` also does not work because loadtest does not return anything. Fix in https://github.com/os-autoinst/os-autoinst/pull/1797